tests/metrics/run_docker_metrics

```diff
@@ -1,24 +1,19 @@
 #!/bin/bash
 
 source ./run_docker_metrics.dat
+
+# Verify that test-common.bash is generated.
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+[ -f "${SCRIPT_PATH}/../lib/test-common.bash" ] || \
```

Comment:
Hi again. That still doesn't seem to work for me. I think what is happening is that the (xxx) starts a subshell, so the 'exit 1' exits that subshell only, and not the 'app'. What I think you need to do is replace the () with {} to create 'group command', like:

```
        { echo "Required files are not generated, please run 'sudo make metrics-tests' from top level of this repo" \
        && exit 1; }
```

I tried this, and that does work for me.